canvas_modules/common-canvas/src/common-properties/controls/toggletext/toggletext.scss

```diff
@@ -29,6 +29,10 @@
 			border-bottom: 2px solid $support-03;
 		}
 	}
+	button svg {
+		padding-left: $spacing-02;
+		padding-top: $spacing-02;
+	}
```

Comment:
You can remove entire `button svg {}` code.
Add following CSS under `.properties-table-cell-control` -

```
// toggle control in table needs to fit within column
.properties-table-cell-control {
	.properties-toggletext button {
		padding: 0 $spacing-02;
	}
	.properties-toggletext svg { // this is new CSS
		margin-left: $spacing-02;
	}
}
```